src/views/RichWorkspace.vue

```diff
@@ -256,42 +225,42 @@ export default {
 		color: var(--color-text-maxcontrast);
 	}
 
-	#rich-workspace:deep(div[contenteditable=false]){
+	#rich-workspace div[contenteditable=false] {
```

Comment:
What is the reason for that change? Should not be needed

Reply:
I noticed that they were not applied to the editor and I was not able to work with the editor normally.

Reply:
Can you maybe check if that works after running `npm ci` I think this might only be the case if you have some outdated dependencies.

Reply:
If that is still an issue we should still address it in a separate PR I think.

Reply:
Ok, let me check after npm ci.

Reply:
@juliushaertl It actually doesn't work.

Reply:
> If that is still an issue we should still address it in a separate PR I think.

I will remove the changes from the current PR and open a new one for the fix.

Reply:
Removed the changes for now.